README.md

```diff
@@ -9,11 +9,21 @@ Install dependencies
 ```
 $ make install
 ```
-Start server at port 8000
+### Start virtual environment:
+    source venv/bin/activate
```

Comment:
You don't need to add it before the make watch. That works without starting the venv, because it is using the python from the venv folder.
https://github.com/liqd/digitalstrategie/blob/89717a0da2283fc5fc410e9e02eaecf28fc59d5c/Makefile#L49

I thought, you just add it before `python manage.py createsuperuser` because you cannot use the python there without starting the virtual env.